recipes/karchive/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conan.tools.files import rename
+from conans import ConanFile, CMake, tools
+import shutil
+
+
+class KArchiveConan(ConanFile):
+    name = "karchive"
+    license = ("BSD-2-Clause", "LGPL-2.0-only", "LGPL-2.0-or-later", "LGPL-3.0-only", "LicenseRef-KDE-Accepted-LGPL")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://api.kde.org/frameworks/karchive/html/index.html"
+    topics = ("archive", "compression")
+    description = "KArchive provides classes for easy reading, creation and manipulation of archive formats like ZIP and TAR."
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "with_lzma": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_lzma": True
+    }
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
seems like it uses `find_package` to find `qt5`

Reply:
@SSE4 I was trying to use cmakedeps with cmaketoolchain but I was not able to get it to work. I will give it a shot with your suggestion